client/src/actions/index.js

```diff
@@ -12,3 +19,60 @@ export const setConversation = id => ({
     type: SET_CONVERSATION,
     id,
 });
+
+export const addBusiness = (storeName, avatar, storeOwner, location, description,
+    needsHelp, tags) => ({
+        type: ADD_BUSINESS,
+        storeName,
+        avatar,
+        storeOwner,
+        location,
+        description,
+        needsHelp,
+        tags,
+    });
```

Comment:
Nitpicking here, but instead of passing in multiple parameters we could do:
```suggestion
export const addBusiness = business => ({
        type: ADD_BUSINESS,
        business,
    });
```
and `business` object would have `storeName`, `avatar`, etc.